packages/date-picker/test/keyboard-input.common.js

```diff
@@ -589,194 +502,5 @@ describe('keyboard', () => {
       expect(focusedDate().getMonth()).to.equal(0);
       expect(focusedDate().getDate()).to.equal(20);
     });
-
-    it('should set datePicker value on blur', async () => {
-      await sendKeys({ type: '1/1/2000' });
-      await sendKeys({ press: 'Tab' });
-      expect(datePicker.value).to.equal('2000-01-01');
-    });
-
-    it('should revert input value on Esc when overlay not initialized', async () => {
-      await sendKeys({ type: '1/1/2000' });
-      await sendKeys({ press: 'Escape' });
-      expect(input.value).to.equal('');
-      expect(datePicker.value).to.equal('');
-    });
-
-    it('should revert input value on Esc when overlay has been initialized', async () => {
-      await open(datePicker);
-      await close(datePicker);
-      await sendKeys({ type: '1/1/2000' });
-      await sendKeys({ press: 'Escape' });
-      expect(datePicker.value).to.equal('');
-    });
```

Comment:
**note:** Long time ago, date-picker had a [specific logic](https://github.com/vaadin/web-components/commit/da05c9c891dcc671f7ad4cb82a4eef6f4cb77a9d#diff-10462f55d251acfc2ad7ddd2db35cee88e15829f67f5824233c7a267cd98a1adR430) that checked for overlay initialization before closing the overlay. I didn't find this logic in the modern version, so I removed these tests completely.